modules/tinymce/src/plugins/autoresize/main/ts/core/Resize.ts

```diff
@@ -134,16 +141,44 @@ const setup = (editor: Editor, oldSize: Cell<number>): void => {
       height: 'auto'
     });
 
-    dom.setStyles(editor.getBody(), {
-      'paddingLeft': overflowPadding,
-      'paddingRight': overflowPadding,
-      // IE & Edge have a min height of 150px by default on the body, so override that
-      'min-height': 0
-    });
+    if (Env.browser.isEdge() || Env.browser.isIE()) {
+      dom.setStyles(editor.getBody(), {
+        'paddingLeft': overflowPadding,
+        'paddingRight': overflowPadding,
+        // IE & Edge have a min height of 150px by default on the body, so override that
+        'min-height': 0
+      });
+    } else {
+      dom.setStyles(editor.getBody(), {
+        paddingLeft: overflowPadding,
+        paddingRight: overflowPadding
+      });
+    }
+
+    resize(editor, oldSize, e, getExtraMarginBottom);
+    resizeCounter += 1;
   });
 
   editor.on('NodeChange SetContent keyup FullscreenStateChanged ResizeContent', (e) => {
-    resize(editor, oldSize, e);
+    if (resizeCounter === 1) {
+      sizeAfterFirstResize = editor.getContainer().offsetHeight;
+      resize(editor, oldSize, e, getExtraMarginBottom);
+      resizeCounter += 1;
+    } else if (resizeCounter >= 2 && !checkDone) {
+      const dom = editor.dom;
+      const doc = editor.getDoc();
+      const isLooping = sizeAfterFirstResize < editor.getContainer().offsetHeight;
+      if (isLooping) {
+        dom.setStyles(doc.documentElement, { 'min-height': 0 });
+        dom.setStyles(editor.getBody(), { 'min-height': 'inherit' });
+      }
+      getExtraMarginBottom = isLooping ? Fun.constant(0) : getExtraMarginBottom;
+      checkDone = true;
```

Comment:
Is the checkDone state really needed here since the resizeCounter could be used just add a else and increment the counter here as well.
```

Reply:
ok, I refactored removing the `checkDone`